options/linux/generic/capabilities.cpp

```diff
@@ -0,0 +1,15 @@
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int capset(void *, void *) {
```

Comment:
These functions don't seem to be part of libc, at least on my host. They aren't declared anywhere, either.

Reply:
they're part of the linux capabilities API, should be declared in `sys/capability.h`

Reply:
The headers seems to be part of `libcap` though.

Reply:
they would have to be implemented as syscalls I think - implementing them here and providing an empty libcap object seems like the nicest option to me

Reply:
I followed musl here, which does this and then calls the linux syscall.

Reply:
See [this](https://github.com/bminor/musl/blob/7d756e1c04de6eb3f2b3d3e1141a218bb329fcfb/src/linux/cap.c#L8).

Reply:
If these are stubbed then they should at least log a message that they are so.

Reply:
As requested, I've added a message.